src/ruby/lib/grpc/core/call.rb

```diff
@@ -0,0 +1,20 @@
+module GRPC
+  module Core
+    class Call
+      # call-seq:
+      # metadata = call.metadata
+      #
+      # Gets the metadata object saved the call.
+      attr_reader :metadata
+
+      # call-seq:
+      # call.metadata = metadata
+      #
+      # Saves the metadata hash on the call.
+      def metadata=(val)
```

Comment:
Is this change for readability? A little unsure of the benefit since the `metadata` setter isn't intended as a part of the surface API.